docs/data/product/dea-fractional-cover-landsat/_data.yaml

```
@@ -6,7 +6,7 @@
 title: DEA Fractional Cover (Landsat)
 long_title: Geoscience Australia Landsat Fractional Cover Collection 3
 header_image: /_files/cmi/FractionalCover1.jpg
-version: 2.3.1
+version: Landsat 5/7: 2.5.0, Landsat8/9: 2.5.1
```

Using quotes so that the YAML formatting will work.

```suggestion
version: "Landsat 5/7: 2.5.0, Landsat8/9: 2.5.1"
```

---

Done, thanks Benji!